docs/api/models.rst

```diff
@@ -146,11 +146,19 @@ Guild
 .. autoclass:: Template()
     :members:
 
+AutoMod
+~~~~~~~
+
 .. attributetable:: AutoModRule
 
 .. autoclass:: AutoModRule()
     :members:
 
+.. attributetable:: AutoModAction
+
+.. autoclass:: AutoModAction()
+    :members:
+
```

Comment:
Please also add `AutoModTriggerMetadata` and `AutoModActionMetadata`

Reply:
Added!